examples/create-window/main.go

```diff
@@ -96,12 +96,22 @@ func main() {
 			fmt.Println("Both event and error are nil. Exiting...")
 			return
 		}
-
+		
 		if ev != nil {
 			fmt.Printf("Event: %s\n", ev)
 		}
 		if xerr != nil {
 			fmt.Printf("Error: %s\n", xerr)
 		}
+		
+		// This is how accepting events work:
+		// The application checks what event we got
+		// (the event must be registered using either xproto.CreateWindow (see l.35) or
+		//  xproto.ChangeWindowAttributes (see l.50)).
+		// and reacts to it accordingly. All events are defined in the xproto subpackage.
+		switch ev.(type) {
+		case xproto.DestroyNotifyEvent:
```

Comment:
Someone might wonder, why the DestroyNotifyEvent won't trigger if the window is closed by user (at least on Ubuntu). Does it trigger on window close for you?

I see, you mentioned it in the comment above (must be registered...), but is it ok to have unreachable code in example? I mean If we show B (how to catch events), don't we have to show A (how to register)?